specification/k3po.control/src/main/scripts/org/kaazing/specification/control/accept.error.if.no.prepare.rpt

```diff
@@ -0,0 +1,68 @@
+#
+# Copyright 2007-2015, Kaazing Corporation. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+accept tcp://localhost:12345
+accepted
+connected
+
+read "START\n"
+read "\n"
+
+write "ERROR\n"
```

Comment:
I would suggest that we change the SPEC, that an ERROR event will trigger the disconnect of the TCP connection and the client must start a new connection if it wishes to continue.  This allows us to tidy clean up, and not get into the error loops mentioned here